de_web_plugin.cpp

```diff
@@ -4814,23 +4899,34 @@ void DeRestPluginPrivate::checkSensorButtonEvent(Sensor *sensor, const deCONZ::A
                         ok = true;
                     }
                 }
+                else if (ind.clusterId() == ADEO_CLUSTER_ID)
+                {
+                    ok = false;
+                    
+                    if (buttonMap.zclParam0 == pl0)
+                    {
+                        ok = true;
+                    }
+                }
                 else if (ind.clusterId() == ADUROLIGHT_CLUSTER_ID)
                 {
                     ok = false;
 
                     if (buttonMap.zclParam0 == zclFrame.payload().at(1))
+
                     {
                         ok = true;
                     }
                 }
 
                 if (ok && buttonMap.button != 0)
                 {
-                    if (!buttonMap.name.isEmpty()) { cmd = buttonMap.name; }
-
-                    DBG_Printf(DBG_INFO, "[INFO] - Button %u - %s%s, endpoint: 0x%02X, cluster: %s, action: %s, payload: %s, zclSeq: %u\n",
-                        buttonMap.button, qPrintable(sensor->modelId()), qPrintable(addressMode), ind.srcEndpoint(), qPrintable(cluster), qPrintable(cmd), qPrintable(zclPayload), zclFrame.sequenceNumber());
-
+                    QString action = QLatin1String("Not defined");
+                    if (!buttonMap.name.isEmpty()) { action = buttonMap.name; }
+                    
+                    DBG_Printf(DBG_INFO, "[INFO] - Button %u - %s%s, endpoint: 0x%02X, cluster: %s, command: %s, action: %s, payload: %s, zclSeq: %u\n",
+                        buttonMap.button, qPrintable(sensor->modelId()), qPrintable(addressMode), ind.srcEndpoint(), qPrintable(cluster), qPrintable(cmd), qPrintable(action), qPrintable(zclPayload), zclFrame.sequenceNumber());
```

Comment:
Not sure if this is good, `cmd` is printed later on in various places. @SwoopX  any thoughts?

Reply:
Can too update the value after the logging.
But change it before can suppress usefull informations, for this device I needed this value for exemple.

Reply:
Fck, pressed bad button

Reply:
Ah, missed that one. Tbh, I don't really get that change. If it's about the term "action", then I'm all in to change it to "command". If my eyes don't deceive me, "action" is used nowhere throughout the function...

Reply:
Nope only used for this debug message.